AirCasting/CreateSessionViews/PowerABView.swift

```diff
@@ -49,17 +50,9 @@ struct PowerABView: View {
     }
 
     var continueButton: some View {
-        NavigationLink(destination: SelectPeripheralView(SDClearingRouteProcess: false, creatingSessionFlowContinues: $creatingSessionFlowContinues, urlProvider: urlProvider)) {
+        NavigationLink(destination: SelectPeripheralView(SDClearingRouteProcess: false, creatingSessionFlowContinues: $creatingSessionFlowContinues, urlProvider: urlProvider, locationHandler: locationHandler)) {
             Text(Strings.Commons.continue)
                 .frame(maxWidth: .infinity)
         }
     }
 }
-
-#if DEBUG
```

Comment:
Could you keep the preview please?

Reply:
I would like to delete it, as I talked to Paweł and after his merge of the architectural change, most of the previews will be gone from the app

Reply:
Previews are really annoying to maintain at this point, we should remove all of them and re-introduce using more structured approach (if necessary), sharing Mock/Dummy data providers with unit test target.